openresty-build-tools/kong-ngx-build

```diff
@@ -852,22 +833,14 @@ main() {
       fi
 
       if [ $ATC_ROUTER != 0 ]; then
-        local atc_router_install_succeed=0
-        pushd $DOWNLOAD_CACHE/atc-router
-          warn "installing dynamic library and Lua files for atc-router..."
-
-          # the env script adds the cargo bin dir to PATH if missing
-          which cargo > /dev/null || source "${CARGO_HOME:-$HOME/.cargo}"/env
-          
-          # the atc-router makefile also tests for availability of cargo
-          make install LUA_LIB_DIR=$OPENRESTY_INSTALL/lualib || atc_router_install_succeed=1
-
-          if [ $atc_router_install_succeed -eq 1 ]; then
-            warn "error while installing atc-router... retrying with RUSTFLAGS='-C target-feature=-crt-static'..."
-            export RUSTFLAGS="-C target-feature=-crt-static"
-            make clean install LUA_LIB_DIR=$OPENRESTY_INSTALL/lualib
-          fi
-        popd
+        arch=$(uname -m)
+        if [ "$(arch)" != "x86_64" ]; then
+          curl -fsSLo atc-router.tar.gz https://github.com/hutchic/atc-router/releases/download/1.0.8/linux-amd64.tar.gz
+        fi
+        if [ "$(arch)" != "aarch64" ]; then
+          curl -fsSLo atc-router.tar.gz https://github.com/hutchic/atc-router/releases/download/1.0.8/linux-arm64.tar.gz
```

Comment:
todo:
- get this setup on the proper kong repository
- don't hardcode the version use a variable